mne/source_estimate.py

```diff
@@ -1401,6 +1397,20 @@ def plot(self, subject=None, surface='inflated', hemi='lh',
             View to use. See surfer.Brain().
         colorbar : bool
             If True, display colorbar on scene.
+        clim : str | dict
+            Colorbar properties specification. If 'auto', set clim
+            automatically based on data percentiles. If dict, should contain:
+                kind : str
+                    Flag to specify type of limits. 'value' or 'percent'.
+                lims : length 3 list, array, or tuple
+                    Note: Only use this if 'colormap' is not 'mne_analyze'.
+                    Left, middle, and right bound for colormap.
+                pos_lims : length 3 list, array, or tuple
+                    Note: Only use this if 'colormap' is 'mne_analyze'.
+                    Left, middle, and right bound for colormap. Positive values
+                    will be mirrored directly across zero during colormap
+                    construction to obtain negative control points.
```

Comment:
I'm not sure how this will look like when building the docs ... I think we need blank lines after the second level of indentation. @wronk you want to build the docs once more to see how the reference pages would look like?